content/applications/finance/fiscal_localizations/chile.rst

```diff
@@ -39,12 +39,10 @@ localization.
      - Adds the *Propuesta F29* and *Balance Tributario (8 columnas)* reports.
    * - :guilabel:`Chile - E-invoicing`
      - `l10n_cl_edi`
-     - Includes all technical and functional requirements to generate and receive electronic
-       invoices online based on the :abbr:`SII (Servicio de Impuestos Internos)` regulations.
-   * - :guilabel:`Chile - Electronic Receipt`
-     - `l10n_cl_boletas`
-     - Includes all technical and functional requirements to generate and receive electronic
-       invoices digitally based on the :abbr:`SII (Servicio de Impuestos Internos)` regulations.
+     - Includes all technical and functional requirements to receive and generate electronic
+       receipts and invoices online based on the :abbr:`SII (Servicio de Impuestos Internos)`
+       regulations. Previous versions had two different modules, now it is merged into this main
+       one, required for EDI workflows.
```

Comment:
```suggestion
       regulations.
```
We avoid referring to other versions of the documentation. The goal is that this page should be clear and precise about how Odoo works in _this_ version. The previous sentence makes it clear that this module has all the technical and functional requirements for both invoices and receipts, so we can remove this sentence.

In the previous versions, it says that the Chile - Electronic Receipt module:

> Includes all technical and functional requirements to generate and receive electronic invoices digitally based on the :abbr:`SII (Servicio de Impuestos Internos)` regulations.

Should that say _receipts_ instead of _invoices_? If so, we should update versions 16 and 17 to make it clear to users who are still on those versions and to users comparing the version 18 documentation to those previous versions' documentation.

Reply:
1. That's helpful for us to know that there's been confusion on the customer end and they're asking questions about it on help tickets. I try to follow our rules where it makes sense, and if there's a situation where that seems to create more problems that it solves, we can adapt. In this situation, do you think it will be clear enough as it is written now? If not, would it be clear enough if we added some italics or bold for emphasis? For example:

> Includes all technical and functional requirements to receive and generate **electronic receipts and invoices** online based on the SII regulations.

Let me know if that works for you :)

2. I understand, so for this one we'll merge the two modules to reflect the actual change in 17.4 like you suggested. My question wasn't clear, but what I'm trying to ask is this: Before the change in 17.4, when these were two separate modules, was the second one supposed to say "receipts" instead of "invoices"?
![image](https://github.com/user-attachments/assets/94bb26b1-0da4-4ae1-a154-cee53d5d4c09)
It seems like it should say "receipts". If I'm correct, in a separate PR targeting 16.0, we should make that update. We'll forward port it just to saas 17.3 so anyone still on 16 or 17 doesn't get confused.